tests/containers/prepare_hpvs.pm

```diff
@@ -0,0 +1,171 @@
+# SUSE's openQA tests
+#
+# Copyright 2022-2023 SUSE LLC
```

Comment:
Why not 2024?

Reply:
haha, just a copy and paste. I haven't paid any attention to this line. Thanks